src/unsecure/mod.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// This module contains an implementation of a negligible-cost (apart from some ocassional pk copying) trivial
+/// cryptographic signature scheme.
+///
+/// Warning: All schemes in this file are completely unsafe to use in production.
```

Comment:
Besides the safety notice, I would add a directional comment indicating what we intend to use this for (benches). This would give a future auditor an easier time making heads or tails of this.